### PR TITLE
add campaign document upload

### PR DESCRIPTION
**Pull request by domagojlatecki**

### CODE REVIEW

File: src/app/shared/services/blockchain/campaign.service.ts [hunk 1]
```diff
@@ -204,6 +205,22 @@ export class CampaignService {
       this.errorService.handleError(),
     )
   }
+
+  private uploadMultipleDocuments(documents: File[] | undefined): Observable<IPFSDocument[]> {
+    if (documents && documents.length > 0) {
```
Comment (mightymatth) [inline src/app/shared/services/blockchain/campaign.service.ts:210]:
```suggestion
    if (documents?.length > 0) {
```
Why not this? Is there any issue?

File: src/app/campaigns/campaign-edit/campaign-edit.component.html [hunk 2]
```diff
@@ -37,6 +37,19 @@ <h1 class="my-2 text-2xl uppercase">
           <input id="photo" type="file" formControlName="photo" multiple>
         </div>
 
+        <div class="mt-2">
+          <label class="block">Documents</label>
+          <div *ngFor="let document of updateForm.value.documents; let i = index">
+            {{ document.name }}
+            <button (click)="updateForm.value.documents.splice(i, 1); updateForm.markAsDirty()">❌</button>
```
Comment (mightymatth) [inline src/app/campaigns/campaign-edit/campaign-edit.component.html:44]: Can you just make a new function for this click action to be more clear?

**Review by mightymatth — CHANGES_REQUESTED** [review]

File: src/app/shared/services/blockchain/campaign.service.ts (hunk 1 above)
Comment (domagojlatecki) [reply src/app/shared/services/blockchain/campaign.service.ts:210]: Forgot about that feature for a moment 😄

Comment (domagojlatecki) [reply src/app/shared/services/blockchain/campaign.service.ts:210]: Actually, for some reason TypeScript is showing an error for `documents?.length` - error message is that `documents can be undefined`... I have no idea why.

**Review by mightymatth — APPROVED** [review]

**Pull request merged**